README.md

```diff
@@ -1,2 +1,17 @@
-# insight-cv
-A framework for scientific image analysis
+<p align="center">
+  <img src="https://i.imgur.com/vyXUpga.png" width="600">
+   <b>
+    <center> <h1>Insight-cv: A toolkit for scientific image analysis 1</h1> </center>
```

Comment:
```suggestion
    <center> <h1>insight-cv: A toolkit for scientific image analysis</h1> </center>
```

Reply:
I think we should try to use lower case when referring to the package name?

Reply:
yep, I was thinking about it when I added the text next to the logo. Should I change then it? @scotthosking any thoughts?

Reply:
we can always modify the logo in a later PR!

Reply:
ok, I'll leave all lowercase, including the logo, as it seems to be the standard for naming packages.